CONTRIBUTING.md

```diff
@@ -80,7 +80,7 @@ You should only worry about manually running any new integration tests that you
 
 1. Create a new git branch:
      ```shell
-     git checkout -b my-fix-branch master
+     git checkout -b my-fix-branch main
```

Comment:
`<start_point>` (the second argument after `-b`) is optional. After forking and cloning the repository, your `HEAD` should be at the latest commit in the default branch already.

Reply:
Agreed! I can add a comment to specify that it is optional if you are starting from main branch.

Reply:
Or just remove it

Reply:
Done!